schema.sql

```diff
@@ -12,3 +12,30 @@ PRIMARY KEY(id)
 
 ALTER TABLE animals ADD species VARCHAR(50);
 
+
+CREATE TABLE owners(
+  id SERIAL NOT NULL,
+  full_name VARCHAR(200),
+  age INT,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE species(
+  id SERIAL NOT NULL,
+  name VARCHAR(200),
+  PRIMARY KEY(id)
+);
+
+
+
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD COLUMN species_id INT;
+
+ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id); 
```

Comment:
[OPTIONAL] Very well done 👍🏼  , you can also combine these two statments in one query `ALTER TABLE animals ADD species_id int REFERENCES species(id);`